lib/samples/find_closest_facility_from_point/find_closest_facility_from_point_sample.dart

```diff
@@ -37,46 +38,61 @@ class _FindClosestFacilityFromPointSampleState
       'https://services2.arcgis.com/ZQgQTuoyBrtmoGdP/ArcGIS/rest/services/San_Diego_Facilities/FeatureServer/0');
   static final _incidentsLayerUri = Uri.parse(
       'https://services2.arcgis.com/ZQgQTuoyBrtmoGdP/ArcGIS/rest/services/San_Diego_Incidents/FeatureServer/0');
+  // Create a task for the closest facility service.
   final _closestFacilityTask = ClosestFacilityTask.withUrl(Uri.parse(
       'https://sampleserver6.arcgisonline.com/arcgis/rest/services/NetworkAnalysis/SanDiego/NAServer/ClosestFacility'));
-
+  // Create a controller for the map view.
   final _mapViewController = ArcGISMapView.createController();
+  // Create a graphics overlay for the route.
   final _routeGraphicsOverlay = GraphicsOverlay();
-  bool _isRouteSolved = false;
-  bool _isInitialized = false;
+  // Create a flag to track whether the route has been solved.
+  bool _routeSolved = false;
+  bool _ready = false;
```

Comment:
These two should be declared with `var` rather than `bool`.

Also there should be the usual comment before the `_ready`: `  // A flag for when the map view is ready and controls can be used.`